release/internal/config/config.go

```diff
@@ -29,8 +29,8 @@ type Config struct {
 	// OperatorConfig is the configuration for Tigera operator
 	OperatorConfig operator.Config
 
-	// ValidArchs are the OS architectures supported for multi-arch build
-	ValidArchs []string `envconfig:"VALID_ARCHES" default:"amd64,arm64,ppc64le,s390x"`
+	// Arches are the OS architectures supported for multi-arch build
+	Arches []string `envconfig:"ARCHES" default:"amd64,arm64,ppc64le,s390x"`
```

Comment:
`VALID_ARCHES` is based on the old hashreleases

Reply:
In the Calico repo, VALIDARCHES is an existing option that is the output from `ARCHES` and `EXCLUDE_ARCHES`

https://github.com/projectcalico/calico/blob/master/lib.Makefile#L34

So, it might have been correct in the old repo but here VALIDARCHES has a different meaning, and the behavior of this field is more aligned with the existing "ARCHES" config option.